app/controllers/songs_controller.rb

```diff
@@ -1,5 +1,34 @@
 class SongsController < ApplicationController
   def index
+    respond_to do |format|
+      format.json do
+        songs = Song.all
+        total_songs = songs.count
+
+        if params[:sSearch].present?
```

Comment:
I'm guessing jQuery or the gem you are using uses camel case for variable names? Just wondering why you didn't use `s_search` instead.